virus/RedMiteAssociatedHypovirus1-virTemplate.md

```diff
@@ -0,0 +1,117 @@
+
+The second publication is a study conducted by Torish et al from NHGRI/NIH. This study aims to provide insight into the relation between the immune system and various viral pathogens on the skin [Tirosh et al., 2018]. In humans, DOCK8 is a protein that is encoded by the DOCK8 gene [Tirosh et al., 2018]. A rare primary immunodeficiency known as DOCK8 deficiency is caused by a lack of DOCK8 [Tirosh et al., 2018]. DOCK8 regulates the actin cytoskeleton, which is important for the movement of immune cells within tissues [Tirosh et al., 2018]. The accession number associated with the data set is SRR10951073. This data set was collected by extracting RNA from skin and nares from DOCK8-deficient patients. The viral genetic component from this data set was examined by using serratus.io and blast. It was found that Papillomaviridae, Polyomaviridae, Poxviridae, Parvoviridae and Herpesvirida have sequence level matches. Specifically, there was a 100 percent match for Papillomaviridae and Polyomaviridae with the data set. 
+
+And lastly, a study conducted by Gu et al. from Northwest A&F University. This study aimed to study the resistance of Solanum pinnatisectum to Phytophthora infestans. The data set associated with this publication is a RNA-sequence data that contained viral RNA components, specifically Betaflexiviridae_RdRp and RdRP_2 with accession cd23245 and pfam00978 respectively. Betaflexiviridae_RdRp are usually found in positive-sense single-stranded RNA viruses [Gu et al., 2020]. These viruses typically infect plants that possess replication proteins that are similar to those found in alphaviruses [Gu et al., 2020]. Similar to RdRP_4, RdRP_2 are produced as a result of the processing of polyproteins [Gu et al., 2020]. 
+
+An important finding across all datasets was the identification of RdRP enzymes, which play a crucial role in viral replication. As a result of their widespread presence across different host species, Red mite associated hypovirus 1 must be able to adapt to diverse ecological conditions. The three data sets consisting of fungi, human skin cells, and plant tissues contain this viral genetic information indicating that this virus is highly adaptable thus allowing its spread through species. A wide variety of ecosystems are also covered by the datasets. Liu et al. focused their study on fungi which are grown in decayed wood ecosystems. Torish et al. focused on human skin microbiomes and Gu et al. focused on plant pathology. This virus has shown potential of thriving under a wide range of environmental conditions [Elena et al., 2009]. It is possible that multiple host switching events have occurred during the evolution of this virus, as evidenced by the diversity of hosts, ranging from fungi to humans and plants [Elena et al., 2009]. Most likely, this virus's ability to adapt to diverse biological systems is a result of its genetic makeup [Elena et al., 2009]. 
+
+
+### Virus Genome {Q3}
+
+```{r}
```

Comment:
The code is not needed/useful for a markdown presentation file here. Please include and embed an image here instead.